lib/v2/screens/profile_screens/guardians/guardians_tabs/components/my_guardians_tab.dart

```diff
@@ -19,9 +19,12 @@ class MyGuardiansTab extends StatelessWidget {
     return StreamBuilder<List<GuardianModel>>(
         stream: BlocProvider.of<GuardiansBloc>(context).guardians,
         builder: (context, AsyncSnapshot<List<GuardianModel>> snapshot) {
+          int count = 0;
           if (snapshot.hasData) {
             var myGuardians = snapshot.data!.where((element) => element.type == GuardianType.myGuardian);
-
+            myGuardians.forEach((element) {
```

Comment:
nope. This is brute force, archaic, rudimentary. 

myGuardians.where{ }

use the where method.

Reply:
True, Fixed.